std/haxe/io/FPHelper.hx

```diff
@@ -56,7 +56,7 @@ class FPHelper {
 		static var helper = new js.html.DataView(new js.html.ArrayBuffer(8));
 	#end
 
-	#if !(neko || cpp || cs || java || flash || nodejs)
+	#if !(neko || cpp || cs || java || flash || (js && nodejs))
```

Comment:
This shouldn't be necessary with https://github.com/HaxeFoundation/hxnodejs/pull/94?

Reply:
Yes, but it makes CI be green, and some people may not updated their hxnodejs